test_scripts/API/SendLocation/001_SendLocation_success.lua

```diff
@@ -0,0 +1,103 @@
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common_send_location = require('test_scripts/API/SendLocation/common_send_location')
+
+--[[ Local Variables ]]
+local request_params = {
+    longitudeDegrees = 1.1,
+    latitudeDegrees = 1.1,
+    address = {
+        countryName = "countryName",
+        countryCode = "countryName",
+        postalCode = "postalCode",
+        administrativeArea = "administrativeArea",
+        subAdministrativeArea = "subAdministrativeArea",
+        locality = "locality",
+        subLocality = "subLocality",
+        thoroughfare = "thoroughfare",
+        subThoroughfare = "subThoroughfare"
+    },
+    timeStamp = {
+        second = 40,
+        minute = 30,
+        hour = 14,
+        day = 25,
+        month = 5,
+        year = 2017,
+        tz_hour = 5,
+        tz_minute = 30
+    },
+    locationName = "location Name",
+    locationDescription = "location Description",
+    addressLines = 
+    { 
+        "line1",
+        "line2",
+    }, 
+    phoneNumber = "phone Number",
+    deliveryMode = "PROMPT",
+    locationImage = 
+    { 
+        value = "icon.png",
+        imageType = "DYNAMIC",
+    }
+}
+
+--[[ Local Functions ]]
+local function send_location(params, self)
+    local cid = self.mobileSession1:SendRPC("SendLocation", params)
+
+    local deviceID = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+    params.locationImage.value = common_send_location.getPathToSDL() .. "storage/"
+    params.locationImage.value = params.locationImage.value .. common_send_location.getMobileAppId(1) .. "_" .. deviceID
+    params.locationImage.value = params.locationImage.value .. "/icon.png"
+
+
+    EXPECT_HMICALL("Navigation.SendLocation", params)
+    :Do(function(_,data)
+        --hmi side: sending Navigation.SendLocation response
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+
+    self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+local function put_file(file_name, self)
+    local CorIdPutFile = self.mobileSession1:SendRPC(
+      "PutFile",
+      {syncFileName = file_name, fileType = "GRAPHIC_PNG", persistentFile = false, systemFile = false},
+      "files/icon.png")
+
+    self.mobileSession1:ExpectResponse(CorIdPutFile, { success = true, resultCode = "SUCCESS"})
+    :Timeout(10000)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common_send_location.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common_send_location.start)
+runner.Step("RAI, PTU", common_send_location.rai_ptu)
+runner.Step("Activate App", common_send_location.activate_app)
+runner.Step("Upload file", put_file, {"icon.png"})
+
+runner.Title("Test")
+runner.Step("SendLocation - all params ", send_location, { request_params })
```

Comment:
@Itileda Please remove redundant whitespace in description of step:
"SendLocation - all params " -> "SendLocation - all params"

Reply:
@aderiabin , fixed in 31e215e56924b5f2a622306db025cb153b02fa1f